lib/Doctrine/DBAL/Migrations/FileQueryWriter.php

```diff
@@ -51,6 +51,10 @@ public function __construct(string $columnName, string $tableName, ?OutputWriter
 
     /**
      * TODO: move SqlFileWriter's behaviour to this class - and kill it with fire (on the next major release)
+     * @param string $path
```

Comment:
You might as well omit these as the method signature already contains type information.